group-coordinator/src/main/java/org/apache/kafka/coordinator/group/classic/ClassicGroup.java

```diff
@@ -956,6 +956,20 @@ public Optional<OffsetExpirationCondition> offsetExpirationCondition() {
         return Optional.empty();
     }
 
+    /**
+     * The state of this group.
+     *
+     * @return The current state as a lowercase String.
+     */
+    public String stateAsLowerCaseString() {
+        return this.state.toLowerCaseString();
+    }
```

Comment:
We could probably remove this one and inline it.

Reply:
Done.